aixplain/factories/index_factory.py

```diff
@@ -1,16 +1,18 @@
 from aixplain.modules.model.index_model import IndexModel
 from aixplain.factories import ModelFactory
-from aixplain.enums import Function, ResponseStatus, SortBy, SortOrder, OwnershipType, Supplier
+from aixplain.enums import EmbeddingModel, Function, ResponseStatus, SortBy, SortOrder, OwnershipType, Supplier
 from typing import Optional, Text, Union, List, Tuple
 
 
 class IndexFactory(ModelFactory):
     @classmethod
-    def create(cls, name: Text, description: Text) -> IndexModel:
+    def create(
+        cls, name: Text, description: Text, embedding_model: EmbeddingModel = EmbeddingModel.OPENAI_ADA002
+    ) -> IndexModel:
         """Create a new index collection"""
         model = cls.get("66eae6656eb56311f2595011")
```

Comment:
We should make this model id a constant variable as well.

Reply:
done